cli/lib/kontena/cli/stacks/yaml/reader.rb

```diff
@@ -131,7 +131,7 @@ def default_envs_to_options
       def variables
         @variables ||= ::Opto::Group.new(
           internals_interpolated_yaml.fetch('variables', {}).merge(default_envs_to_options),
-          defaults: { from: :env, to: :env }
+          defaults: { from: :env }
```

Comment:
This is a really small change, but does it also have some other side-effects? I don't know myself... I'd rather have a separate one-liner PR for this change than having to think about it here and now before being able to merge this.